components/InputFormGuest.js

```diff
@@ -11,11 +11,57 @@ export default function AddGuest() {
           <Input
             className="SmallFontStyle"
             type="text"
-            id="contactData"
-            name="contactData"
+            id="firstName"
+            name="firstName"
             required
+            placeholder="Vorname"
+          />
+          <Input
+            className="SmallFontStyle"
```

Comment:
Why do you use a className here and for the other elements? What's its purpose?

Reply:
Thanks for your review!

Because of the font. All my fonts are deposited in "global.css". Is there another option to change the font instead of using a className?